### ISSUE TITLE
508-defect-2: DOM order for contact on the contact information page should be logically ordered

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#53448]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

Currently the DOM order has the heading, then the edit button, then the # or address. Ideally we'd want the screen reader user to hear their mobile phone number out before deciding to edit it.

### Reproduction steps

1. Turn on your screen reader on the contact information page.
2. Navigate line by line and confirm that "edit" comes before the item they are actually editing.

### Proposed solution or next steps

The DOM order largely determines the order in which it is read out by assistive technology. To ensure that screen reader users hear their mobile phone number first before deciding to edit it, we'll need to make sure that it comes _before_ the edit link in the DOM.

<img width="1375" alt="Screenshot 2023-02-09 at 3 55 28 PM" src="https://user-images.githubusercontent.com/14154792/217937097-1a800ff0-846d-4219-9ff3-056e0a397c5f.png">

Doing so may impact the visual design (as seen above). @eileen-coforma for awareness ❇️ 

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
This work has been merged and is ready for re-testing.

**Comment by joshkimux** [comment]
<img width="1532" alt="Screenshot 2023-02-21 at 1 14 59 PM" src="https://user-images.githubusercontent.com/14154792/220426681-b43b78a9-2a40-42f2-bc38-921b8ed11893.png">

Excellent! Confirming this has been fixed
